service/gcs/bridge/bridge.go

```diff
@@ -365,10 +425,17 @@ func (b *Bridge) createContainer(w ResponseWriter, r *Request) {
 		b.PublishNotification(notification)
 	}()
 
+	// Set our protocol selected version before return.
+	b.protVer = prot.PvV3
 	w.Write(response)
 }
 
 func (b *Bridge) execProcess(w ResponseWriter, r *Request) {
+	if r.Version != prot.PvV3 {
+		w.Error("", gcserr.NewHresultError(gcserr.HrVmcomputeUnsupportedProtocolVersion))
+		return
+	}
+
```

Comment:
This above checking purposely rejects anything that's not V3.
What protocol version does the RS4 HCS use? V4?

Reply:
That check was removed in a later commit in this PR. Also, yeah, the RS4 HCS will use V4. We're currently working on the V4 stuff in the separate `multicontainer` branch, so that we don't break anything in master until it's all ready to be merged in.